liblangutil/Scanner.cpp

```diff
@@ -306,9 +306,10 @@ bool Scanner::tryScanEndOfLine()
 	return false;
 }
 
-Token Scanner::scanSingleLineDocComment()
+int Scanner::scanSingleLineDocComment()
 {
 	LiteralScope literal(this, LITERAL_TYPE_COMMENT);
+	int endPosition = 0;
```

Comment:
This will return zero if there is `///` at the end of the source. Shouldn't you set `endPosition = m_source->position();` before entering the while loop?

Reply:
Ah, true. Good catch! Thanks.